scripts/all-tests.sh

```diff
@@ -12,7 +12,7 @@ set -e
 
 # Violations thresholds for failing the build
 export PYLINT_THRESHOLD=4175
-export JSHINT_THRESHOLD=7550
+export ESLINT_THRESHOLD=64345
```

Comment:
Why add exceptions? Having a high number is ok, if it's an accurate reflection of what we want to fix eventually.